packages/hydrogen/src/customer/customer.ts

```diff
@@ -480,13 +480,16 @@ function ifInvalidCredentialThrowError(
     new URL(customerAccountUrl);
   } catch {
     console.error(
-      'You do not have the valid credential to use Customer Account API.\nRun `npx shopify hydrogen env pull` to link your store credentials.',
+      new Error(
+        '[h2:error:customerAccount] You do not have the valid credential to use Customer Account API.\nRun `h2 env pull` to link your store credentials.',
+      ),
     );
 
-    throw new Response(
-      'You do not have the valid credential to use Customer Account API (/account).' +
-        '\n\nmock.shop does not support Customer Account API. \nWe recommend running `npx shopify hydrogen env pull` to link your store credentials.',
-      {status: 500},
-    );
+    const publicMessage =
+      process.env.NODE_ENV === 'production'
+        ? 'Internal Server Error'
+        : 'You do not have the valid credential to use Customer Account API (/account).';
```

Comment:
I think there is a strong preference from UX to show the next step in browser. Ideally even with bullet points formatting. 

So lets keep the next step here and remove from the error message above

Reply:
Really? It's kinda weird to me to show terminal commands in the browser but don't show them in the actual terminal.
Also, we can't know what's the proper command (`h2`, `npx shopify hydrogen`, `yarn shopify hydrogen`, etc.) in the browser but we know it in the CLI.

@mynameisadamf @benjaminsehl Do you think it's OK if we only show `You do not have the valid credential to use Customer Account API (/account).` in the browser when the developer forgets to setup CAAPI, and a more informative error in the terminal `You do not have the valid credential to use Customer Account API.\n Run 'h2 env pull' to link your store credentials.` ?

Like the following:

<img width="796" alt="image" src="https://github.com/Shopify/hydrogen/assets/1634092/cab52cd2-4a15-4b34-9321-e070a226c3bc">

Reply:
Yeah it's a general catch all, so better skip the "mock.shop" message in the browser I guess.